gh-md-toc

```diff
@@ -235,23 +236,23 @@ gh_toc_grab() {
                          }
                          modified_href = modified_href res
                     }
-                    print sprintf("%*s", (level-1)*'"$2"', "") "* [" text "](" gh_url  modified_href ")"
+                    print sprintf("%*s", (level)*'"$2"', "") "* [" text "](" gh_url  modified_href ")"
                     '
     if [ `uname -s` == "OS/390" ]; then
         grepcmd="pcregrep -o"
         echoargs=""
         awkscript='{
-                     level = substr($0, length($0), 1)
-                     text = substr($0, match($0, /a>.*<\/h/)+2, RLENGTH-5)
+                     level = substr($0, 3, 1)
+                     text = substr($0, match($0, />[^<]*<span aria-hidden/)+1, RLENGTH-17)
                      href = substr($0, match($0, "href=\"([^\"]+)?\"")+6, RLENGTH-7)
                      '"$common_awk_script"'
                 }'
     else
         grepcmd="grep -Eo"
         echoargs="-e"
         awkscript='{
-                     level = substr($0, length($0), 1)
-                     text = substr($0, match($0, /a>.*<\/h/)+2, RLENGTH-5)
+                     level = substr($0, 3, 1)
+                     text = substr($0, match($0, />[^<]*<span aria-hidden/)+1, RLENGTH-17)
```

Comment:
I'm not sure why, but sometimes I get a character added/chopped and have to adjust this.

Reply:
I think it was to do with headings having emoji 🙃

Reply:
Just out of interest, how did you narrow it down to this?

Reply:
I looked at the HTML returned from the GitHub raw markdown API and adjust the awk commands accordingly, based on where in the HTML the things being matched were. I have no idea if the match strings I have changed to will work for everything; I'm not at all familiar with the GitHub API!

Reply:
Ah I got it now; thanks for explaining!